postman/AskDarcel%20API.postman_collection.json

```diff
@@ -2656,6 +2656,166 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Add Breaking News",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Adding document\", function () {",
```

Comment:
I think we want to change "document" to "news article" or whatever on this line and in any instances below